pkg/manager/member/tidb_member_manager.go

```diff
@@ -390,12 +390,24 @@ func getTiDBConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 	}
 
 	plugins := tc.Spec.TiDB.Plugins
-	startScript, err := RenderTiDBStartScript(&TidbStartScriptModel{
-		ClusterName:     tc.Name,
+	tidbStartScriptModel := &TidbStartScriptModel{
 		EnablePlugin:    len(plugins) > 0,
 		PluginDirectory: "/plugins",
 		PluginList:      strings.Join(plugins, ","),
-	})
+	}
+
+	if len(tc.Spec.PDAddress) > 0 {
```

Comment:
I think if both `spec.pd` and `spec.pdAddress` are configured, the `<cluster>-pd` address should take precedence over `spec.pdAddress`.

Reply:
done